src/wbase/Task.cc

```diff
@@ -135,6 +147,23 @@ Task::Task(TaskMsgPtr const& t, std::string const& query, int fragmentNumber,
     _scanInfo.scanRating = msg->scanpriority();
     _scanInfo.sortTablesSlowestFirst();
     _scanInteractive = msg->scaninteractive();
+
+    /* &&&
+    /// For all of the histograms, all entries should be kept at least until the work is finished.
+    std::chrono::milliseconds maxAge(48h);
+    histSizePerChunk = util::Histogram::Ptr(new util::Histogram(
+            string("SizePerChunk") + _idStr, {1'000, 10'0000, 1'000'000, 10'000'000, 100'000'000}, maxAge,
```

Comment:
The binning of the histogram is not intuitively clear. here is what you have:
```
1'000
10'0000    <-- this is probably a mistake. Should it be 10'000 instead?
1'000'000
10'000'000
100'000'000
```
I would make the histogram binning to be strictly exponential, like:
```
1'000
10'000
100'000
1'000'000
10'000'000
100'000'000
```